sites/shared/components/sets/set-candidate.mjs

```diff
@@ -35,32 +43,31 @@ export const SetLacksMeasies = ({ set, design, t, href, clickHandler }) => {
   return <Component {...wrapProps}>{inner}</Component>
 }
 
-export const SetSummary = ({ set, href, clickHandler }) => {
-  const wrapProps = {
-    icon: <OkIcon className="w-10 h-10 text-success" />,
-    title: <Title set={set} />,
-  }
-  if (clickHandler) wrapProps.onClick = () => clickHandler(set)
-  else if (href) wrapProps.href = href
-
-  const Component = clickHandler ? ChoiceButton : ChoiceLink
-
-  return <Component {...wrapProps}></Component>
-}
-
-export const SetCandidate = ({ set, design, requiredMeasies = [], href, clickHandler }) => {
-  const { t } = useTranslation(['sets'])
-
-  const setProps = { set, design, t, href, clickHandler }
+export const SetCandidate = ({
+  set,
+  design,
+  requiredMeasies = [],
+  href,
+  clickHandler,
+  language,
+}) => {
+  const { t } = useTranslation(['sets', design])
 
+  let hasMeasies = true
+  const setMeasies = set.measies && Object.keys(set.measies)
   // Quick check for required measurements
-  if (!set.measies || Object.keys(set.measies).length < requiredMeasies.length)
-    return <SetLacksMeasies {...setProps} />
+  if (!set.measies || setMeasies.length < requiredMeasies.length) hasMeasies = false
 
   // Proper check for required measurements
-  for (const m of requiredMeasies) {
-    if (!Object.keys(set.measies).includes(m)) return <SetLacksMeasies {...setProps} />
+  if (hasMeasies) {
```

Comment:
Please use the `hasRequiredMeasurements` method in `sites/shared/utils.mjs` here.

Reply:
Note: regarding me remarks below. This method will also return the missing measurements, which is what people want to see.